game_of_thrones/app/views/characters/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2> New character </h2>
+<%= form_for  [@house, @character] do |f|  %>
+
+<p>
+  <%= f.label :character_name%>
+  <%= f.text_field :name%>
+</p>
+<p>
+  <%= f.label :Character_photo%>
+  <%= f.text_field :image_url%>
+</p>
+
+<p>
+  <%= f.submit%>
```

Comment:
ERB code is more readable when you include a space before the closing ERB tag (i.e., `%>`)